src/main/resources/org/schemaspy/types/mssql.properties

```diff
@@ -38,3 +38,7 @@ FROM sysobjects cnstrs \
 INNER JOIN sysobjects tbls ON cnstrs.parent_obj = tbls.id \
 INNER JOIN syscomments cmts ON cmts.id = cnstrs.id \
 WHERE cnstrs.type = 'C' 
+
+selectRoutinesSql=SELECT routine_name, routine_type, data_type AS dtd_identifier, routine_body, routine_definition, is_deterministic, sql_data_access, NULL AS security_type, NULL AS sql_mode, NULL AS routine_comment FROM information_schema.routines WHERE routine_schema = :schema
+
+selectRoutineParametersSql=SELECT specific_name, parameter_name, data_type AS dtd_identifier, parameter_mode FROM information_schema.parameters WHERE specific_schema = :schema AND ordinal_position != 0 ORDER BY ordinal_position
```

Comment:
`information_schema` is not supported prior to SqlServer 2012.

Should a new properties file be added for 2012? (this would require 3 - one for the sql driver, and two for jtds)